src/record/api.rs

```diff
@@ -317,7 +364,11 @@ mod tests {
     assert_eq!(format!("{}", Row::Double(6.1234)), "6.1234");
     assert_eq!(format!("{}", Row::Str("abc".to_string())), "\"abc\"");
     assert_eq!(format!("{}", Row::Bytes(ByteArray::from(vec![1, 2, 3]))), "[1, 2, 3]");
-    assert_eq!(format!("{}", Row::Timestamp(12345678)), "12345678");
+    assert_eq!(format!("{}", Row::Date(14611)), convert_date_to_string(14611));
+    assert_eq!(
+      format!("{}", Row::Timestamp(1262391174000)),
+      convert_timestamp_to_string(1262391174000)
```

Comment:
nit: can we have separate tests for `convert_date_to_string` and `convert_timestamp_to_string`?

Reply:
I added separate tests for new methods. Plus, these ones for general display.